src/Controller/EnMarche/AdherentController.php

```diff
@@ -201,6 +201,20 @@ public function createCitizenProjectAction(Request $request, TurnkeyProject $tur
         ]);
     }
 
+    /**
+     * @Route("/mes-comites", name="app_adherent_committees", methods={"GET"})
+     */
+    public function committeesAction(Request $request): Response
+    {
+        $manager = $this->get('app.committee.manager');
```

Comment:
You can inject this service in this action instead:

`public function committeesAction(Request $request, CommitteeManager $manager): Response`